pkg/controller/compliance/compliance_controller.go

```diff
@@ -184,11 +185,18 @@ func (r *ReconcileCompliance) Reconcile(ctx context.Context, request reconcile.R
 		return reconcile.Result{}, err
 	}
 
-	if err = utils.CheckLicenseKey(ctx, r.client); err != nil {
+	license, err := utils.FetchLicenseKey(ctx, r.client)
+	if err != nil {
 		r.status.SetDegraded("License not found", err.Error())
 		return reconcile.Result{RequeueAfter: 10 * time.Second}, nil
 	}
 
+	if !utils.IsFeatureActive(license, common.ComplianceFeature) {
+		log.Info("Compliance is not activated as part of this license")
```

Comment:
This will potentially be a very chatty log, if we're logging it every 10 seconds. Might want to make it a debug log?